test/indexer/data/59fb2c7b-b699-a97b-bf44-1d51b03a546d.metadata.json

```diff
@@ -1,103 +1,110 @@
 {
-    "links": [
```

Comment:
Question: Judging by the diffs of the cans, we still seem to have an issue with determinism in those. For example, why did the `links` entry move?

Reply:
Decided at PL to sort entities before populating the bundle.